wiki/Tournaments/oPHST/2018/en.md

```diff
@@ -0,0 +1,242 @@
+---
+tags:
+- oPHST 2018
+- oPHST2018
+---
+
+osu!Philippines Summer Tournament 2018
+=============================
+
+![oPHST2018 Poster](poster.png)
+
+> The sun is up, temperatures are high, and it's time to set your rhythm ablaze!
+
+The **osu!Philippines Summer Tournament 2018** is a semi-invitational, semi-open Tournament Series hosted by the osu!PH community for the osu!standard, osu!catch, and the osu!mania game modes. This tournament is to be considered the flagship competition of the community and will serve as the Philippine National Championships for the respective game modes. 
+
+This also serves as the third edition of the annual **osu!mania Philippines Summer Tournament**.
+
+## Tournament Schedule
+### osu!standard
+
+| Event | Timestamp |
+|---:|---|
+| Registration Phase   | 2018-04-02/2018-04-15 |
+| Round of 32          | 2018-04-28/2018-04-29 |
+| Round of 16          | 2017-05-05/2017-05-06 |
+| Quarterfinals        | 2017-05-12/2017-05-13 |
+| Semifinals           | 2017-05-19/2017-05-20 |
+| Finals               | 2017-05-26/2017-05-27 |
+
+### osu!catch
+
+| Event | Timestamp |
+|---:|---|
+| Registration Phase   | 2018-04-02/2018-04-15 |
+| Round of 32          | 2018-04-28/2018-04-29 |
+| Round of 16          | 2017-05-05/2017-05-06 |
+| Quarterfinals        | 2017-05-12/2017-05-13 |
+| Semifinals           | 2017-05-19/2017-05-20 |
+| Finals               | 2017-05-26/2017-05-27 |
+
+### osu!mania
+
+| Event | Timestamp |
+|---:|---|
+| Registration Phase   | 2018-04-02/2018-04-15 |
+| Qualifiers (4K)      | 2018-04-28/2018-04-29 |
+| Qualifiers (7K)      | 2017-05-05/2017-05-06 |
+| Group Stage (4K)     | 2017-05-12/2017-05-13 |
+| Group Stage (7K)     | 2017-05-19/2017-05-20 |
+| Quarterfinals        | 2017-05-26/2017-05-27 |
+| Semifinals           | 2017-06-02/2017-06-03 |
+| Finals Week 1        | 2017-06-09/2017-06-10 |
+| Finals Week 1        | 2017-06-06/2017-06-17 |
+
+## Prizes
+
+*Winners of each Tournament will receive a Profile Badge. the rest of the prizes are being decided*

```

Comment:
Tournament -> game mode

the rest of the prizes are being decided -> The rest of the prizes are to be decided